jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -2502,6 +2502,59 @@ YUI.add('juju-env-api', function(Y) {
     },
 
     /**
+      Return the authorized SSH keys for the specified user.
+
+      @method listKeys
+      @param {String} user The user for whom the keys must be listed.
+      @param {Boolean} fullKeys Whether to pass full keys to the callback, or
```

Comment:
fullKeys feels like a list of keys or something, maybe something more boolean centric like justFingerprints or isFingerprint?